airbyte-ci/connectors/pipelines/pipelines/gradle.py

```diff
@@ -37,11 +37,14 @@ def __init__(self, context: PipelineContext) -> None:
         super().__init__(context)
 
     @property
-    def connector_java_build_cache(self) -> CacheVolume:
-        # TODO: remove this once we finish the project to boost source-postgres CI performance.
-        # We should use a static gradle-cache volume name.
-        cache_volume_name = hacks.get_gradle_cache_volume_name(self.context, self.logger)
-        return self.context.dagger_client.cache_volume(cache_volume_name)
+    def persistent_cache_volume(self) -> CacheVolume:
+        """This cache volume is for sharing gradle state across all pipeline runs."""
+        return self.context.dagger_client.cache_volume("gradle-persistent-cache")
+
+    @property
+    def connector_transient_cache_volume(self) -> CacheVolume:
+        """This cache volume is for sharing gradle state across tasks within a single connector pipeline run."""
+        return self.context.dagger_client.cache_volume(f"gradle-{self.context.connector.technical_name}-transient-cache")
```

Comment:
Can we add a pipeline identifier (like the `git_revision`) to the cache volume name?
If dagger ever enable an automatic *cross pipeline run* persistence of the cache volume it will guarantee this volume will be only used for a single commit.

Reply:
I have no objections. We can discuss this offline but I was under the impression that mounting this volume as PRIVATE would be enough. On second thought, perhaps not.